docs/questions/sharing/visualizing-results.md

```diff
@@ -121,11 +121,11 @@ If you have a bar chart like Count of Users by Age, where the x-axis is a number
 
 ![Scatter](../images/scatter.png)
 
-## Pie chart or donut charts
+## Pie, donut, and sunburst chart
 
-A [pie chart or donut chart](./visualizations/pie-or-donut-chart.md) can be used when breaking out a metric by a single dimension, especially when the number of possible breakouts is small, like users by gender.
+A [pie chart or donut chart](./visualizations/pie-or-donut-chart.md) can be used when breaking out a metric by a single dimension, especially when the number of possible breakouts is small, like accounts by plan. A [sunburst chart](./visualizations/pie-or-donut-chart.md) chart is a multi-dimensional pie chart that can be used to a metric with multiple hierarchical breakouts.
```

Comment:
```suggestion
A [pie chart or donut chart](./visualizations/pie-or-donut-chart.md) can be used when breaking out a metric by a single dimension, especially when the number of possible breakouts is small, like accounts by plan.

A [sunburst chart](./visualizations/pie-or-donut-chart.md) chart is a pie chart with more than one ring to show the data broken out by additional dimensions.
```
What do you think of this revision? The original was a bit technical; we can spell out that stuff on the pie chart page.